include/fasp_to_fenics.h

```diff
@@ -27,4 +27,8 @@ dCSRmat EigenMatrix_to_dCSRmat(const dolfin::EigenMatrix* mat_A);
 
 dvector EigenVector_to_dvector(const dolfin::EigenVector* vec_A);
 
+dolfin::EigenVector Copy_dvector_to_EigenVector(const dvector* vec_b);
+
+void Copy_dvector_to_Function(dolfin::Function* F, const dvector* vec_b);
```

Comment:
I think that functions should always start with lower case, except for class names.  This would make this ``copy_dvector_to_Function``.  It's a small point, but I like consistent naming; we should pick some convention and stick to it.  (this also applies to ``copy_dvector_to_EigenVector``).

Since ``EigenVector`` is a class name, ``EigenVector_to_dvector`` is fine.